develop/app/Database/Seeds/Wallet.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Database\Seeds;
+
+use CodeIgniter\Database\Seeder;
+use Faker\Factory;
+
+class Wallet extends Seeder
+{
+    public function run()
+    {
+        $faker = Factory::create();
+        $now   = date("Y-m-d H:i:s");
+
+        for ($i = 1; $i < 51; $i++) {
```

Comment:
Ditto

Reply:
However, we may need the user's balance for payment deduction or compensation.
Do you mean to bring the balance field into the header?